pkg/render/logstorage/linseed/linseed.go

```diff
@@ -185,6 +185,13 @@ func (l *linseed) linseedClusterRole() *rbacv1.ClusterRole {
 			Resources: []string{"managedclusters"},
 			Verbs:     []string{"list", "watch"},
 		},
+		// These permissions are necessary to allow the management cluster to monitor secrets that we want to propagate
+		// through to the managed cluster for identity verification such as the Voltron Linseed public certificate
+		{
+			APIGroups: []string{""},
+			Resources: []string{"secrets"},
+			Verbs:     []string{"get", "list", "watch"},
```

Comment:
I think we can include a `ResourceNames` section here to limit specifically to the secret we want to copy, no? It should always be a well-known secret name + namespace.

Reply:
I think it's OK to do that in parallel and we can merge this to at least get things working. 

It should definitely be possible to have the informer only watch a particular resource, it might require multiple in order to watch multiple resources though.